pkg/http/api_test.go

```diff
@@ -2094,13 +2094,17 @@ func (f *fakeHTTPServer) DoRequest(method, path string, body []byte, params map[
 func TestV1StateEndpoints(t *testing.T) {
 	etag := "`~!@#$%^&*()_+-={}[]|\\:\";'<>?,./'"
 	fakeServer := newFakeHTTPServer()
-	fakeStore := fakeStateStore{}
+	var fakeStore state.Store = fakeStateStore{}
```

Comment:
nit: You can let the `fakeStore` variable remain type `fakeStateStore` and let Go cast it for you in `fakeTransactionalStores`. Sort of along the lines of "Return implementations and accept interfaces".

Reply:
It doesn't. I get a compilation error. Compiler wins!

pkg/http/api_test.go:2102:22: invalid type assertion: fakeStore.("github.com/dapr/components-contrib/state".TransactionalStore) (non-interface type fakeStateStore on left)

Reply:
For future reference (no need to type assertions)
```golang
var fakeStore fakeStateStore
fakeStores := map[string]state.Store{
	"store1": fakeStore,
}
fakeTransactionalStores := map[string]state.TransactionalStore{
	"store1": fakeStore,
}
```